ci-operator/step-registry/openshift-qe/control-plane-smoke/openshift-qe-control-plane-smoke-commands.sh

```diff
@@ -22,12 +22,20 @@ set_pods_per_node(){
 }
 
 set_pods_per_node
+echo "######"
+UUID="$(uuidgen)-cpt"
```

Comment:
And also can we keep the suffix as `s/-cpt/-CPT/g` so that it never clashes with the string generated by uuids

Reply:
Cool, will prefix it with `CPT-`